### PR TITLE
#169678747 CORS

### PR DESCRIPTION
**Pull request by urbanishimwe**
#### What does this PR do?
- enabling CORS in the application
#### Description of Task to be completed?
Providing securities for our server to be followed up, on the frontend
#### How should this be manually tested?
try to make a `curl` http request with `method` other the ones defined in the
file changes
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[story](https://www.pivotaltracker.com/story/show/169678747)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by Crispy-rw — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -13,6 +13,20 @@ const app = express();
 
 dotenv.config();
 
+app.use(cors());
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
```
Comment (swaibat) [inline src/index.js]: great job @urbanishimwe  but basically you have used two methods:
- you have used CORS package
- you have also created cors middleware your self.
so basically choose among the two using a package or implementing it  your self

**Review by swaibat — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (elchroy) [reply src/index.js]: Please stick to one method, preferably the first.

Comment (urbanishimwe) [reply src/index.js]: okay, right away!

File: src/index.js [hunk 2]
```diff
@@ -13,6 +13,16 @@ const app = express();
 
 dotenv.config();
 
+app.use(cors());
+app.use((req, res, next) => {
```
Comment (swaibat) [inline src/index.js]: @urbanishimwe  great job but also the second middleware is also not required when you're using CORS middleware package basically you need to add only `app.use(cors());`

**Review by swaibat — CHANGES_REQUESTED** [review]

**Review by swaibat — APPROVED** [review]

**Pull request merged**